spark/wordcount/src/main/scala/com/example/bigtable/spark/wordcount/WordCount.scala

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *            http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigtable.spark.wordcount
+
+import com.google.cloud.bigtable.hbase.BigtableConfiguration
+import org.apache.hadoop.hbase.{HColumnDescriptor, HTableDescriptor, TableName}
+import org.apache.hadoop.hbase.client.{BufferedMutator, Connection, Put, RetriesExhaustedWithDetailsException}
+import org.apache.hadoop.hbase.util.Bytes
+import org.apache.spark.SparkContext
+
+/**
+  * Basic WordCount sample of using Cloud Dataproc (managed Apache Spark)
+  * to write to Cloud Bigtable.
+  */
+object WordCount {
+  val ColumnFamily = "cf"
+  val ColumnFamilyBytes = Bytes.toBytes(ColumnFamily)
+  val ColumnNameBytes = Bytes.toBytes("Count")
+
+  def createConnection(ProjectId: String, InstanceID: String): Connection = {
+    BigtableConfiguration.connect(ProjectId, InstanceID)
```

Comment:
There are some settings that work well for Batch systems.  [Here](https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/bigtable-client-1.0.0-pre2/bigtable-dataflow-parent/bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableConfiguration.java#L162)'s how we do it for Dataflow.